vector-im/element-android
# Voice Broadcast - Stop recording on app restart

## Type of change


## Content

Stop the recording when opening the room timeline after an application crash or restart
Also, handled already paused VB before pausing again when the timeline is not shown anymore

## Motivation and context

The voice broadcast is stuck in a recording state when the app is not active anymore, until we improve the code to be able to resume a VB from the last sequence, it is better to stop it as soon as we detect the wrong recording state

## Screenshots / GIFs

## Tests

- Start record a voice broadcast
- Kill the app
- Open the app and go to the room where you were recording
- Verify that the VB is automatically stopped

## Tested devices

- OS version(s):

## Checklist

- [x] Pull request is based on the <del>develop</del> resilience-rc branch
